### PR TITLE
Fixed interfaceOrientation deprecated warning

### PR DESCRIPTION
**Pull request by aryaxt**
#116

### CODE REVIEW

**Comment by heefan** [comment]
+1， for iOS 8

File: SlideMenu/Source/SlideNavigationController.m [hunk 1]
```diff
@@ -570,16 +570,18 @@ - (CGRect)initialRectForMenu
         return rect;
     }
 	
-	if (UIInterfaceOrientationIsLandscape(self.interfaceOrientation))
+	UIDeviceOrientation orientation = [UIDevice currentDevice].orientation;
+	
+	if (UIInterfaceOrientationIsLandscape(orientation))
```
Comment (aryaxt) [inline SlideMenu/Source/SlideNavigationController.m]: Does UIInterfaceOrientationIsLandscape work for UIDeviceOrientation?

Comment (jonasman) [inline SlideMenu/Source/SlideNavigationController.m]: UIDeviceOrientationIsLandscape This might be the correct function to check it.

Comment (aryaxt) [inline SlideMenu/Source/SlideNavigationController.m]: Cool, thanks will update and release a new version hopefully this weekend

**Pull request merged**